pages/events/index.tsx

```diff
@@ -1,12 +1,86 @@
 import { Typography } from 'components/atoms/Typography/Typography';
+import { Button } from 'components/atoms/Button/Button';
 import Link from 'next/link';
+import Image from 'next/image';
+import UserIcon from 'components/icons/UserIcon.svg';
+import Ticket from 'components/icons/Ticket.svg';
+import { useGetAllEventsQuery } from 'generated/graphql';
 
 const EventsPage = () => {
+	const { data, loading } = useGetAllEventsQuery();
+	console.log(data, loading);
+	if (loading) {
+		return (
+			<div>
+				<Typography component="h1">Imprezy</Typography>
+				<div className="flex justify-center">
+					<Image
+						src={'/RingSpinner.svg'}
+						width={80}
+						height={80}
+						alt="Spinner"
+					/>
+				</div>
+			</div>
+		);
+	}
+	if (!data) {
+		return (
+			<div>
+				<Typography component="h1">Data not found</Typography>
+			</div>
+		);
+	}
 	return (
-		<>
-			<Typography component="h1">Events dummy page</Typography>
-			<Link href="events/create">Utwórz nową imprezę</Link>
-		</>
+		<div>
+			<Typography component="h1">Imprezy</Typography>
+			<div className=" fixed bottom-0 w-full">
+				<div className="float-right mr-10 opacity-95 md:mr-72">
+					<Button>
```

Comment:
It's not a good practice to keep a link inside a button, if you want a link to look like a button just style it correctly, you can even create a generic component for that

Reply:
Link has own styles applied